lib/rules/jsx-no-literals.js

```diff
@@ -25,23 +25,29 @@ module.exports = {
       properties: {
         noStrings: {
           type: 'boolean'
+        },
+        printNodeValue: {
```

Comment:
i do not think it makes any sense to add this as a configuration option; if it's useful, it should be in the message for everyone.